.gitlab/ci/.gitlab-ci.variables.yml

```diff
@@ -18,7 +18,6 @@ variables:
   ARTIFACTS_FOLDER_XPANSE: "${CI_PROJECT_DIR}/artifacts/xpanse"
   JIRA_ISSUE_UNRESOLVED_TRANSITION_NAME: "Backlog"
   JIRA_PROJECT_ID: "CIAC"
-  JIRA_SERVER_URL: "https://jira-dc.paloaltonetworks.com"
```

Comment:
removing jira server url as it's different per environment and moved to env variables

Reply:
Good call.
Also: It's internal, so no need to be in a public repo.